packages/Liquid-Tests.package/LQObjectRepoTests.class/instance/setUpRemoteServer.st

```diff
@@ -1,7 +1,7 @@
-tests
+running
 setUpRemoteServer
 	|port|
-	port := PortAssigner get.
+	port := LQPortAssigner get.
```

Comment:
[pebble] What happens if tearDown wasn't executed last time? Will there be problems with just overwriting the existing one? I personally think we should generally also cleanup in setUp to be safe, but I don't know what is the recommended way of doing this, I just remember reading it somewhere once. Julian and me also wondered about this while writing tests, what do you think?

Reply:
I don't know about doing cleanup in setUp.
LQPortAssigner is not affected by setUp or tearDown, but just always counts up the Ports, with each `get` it returns the next port in its list. Is this what you meant?

Reply:
Ahh sorry, didn't see the self fail in the last line, so this is purely a cosmetic suggestion now: replace 
[dummy escalate] on: RemoteError do: [^ self "everything is good" ].
	self fail
with
	self
		should: [dummy escalate]
		raise: RemoteError
I tested it, it runs through